# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/no-net-ifnames.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Who would not want their network interfaces named enP5p1s0 and wlP4p1s0, ...

--- systemd-258.2/src/udev/net/link-config.c.vanilla	2025-12-17 12:23:56.491963498 +0100
+++ systemd-258.2/src/udev/net/link-config.c	2025-12-17 12:27:50.761979450 +0100
@@ -724,7 +724,7 @@
         if (r < 0)
                 log_warning_errno(r, "Failed to parse net.ifnames= kernel command line option, ignoring: %m");
         if (r <= 0)
-                return (cached = true);
+                return (cached = false);
 
         if (!b)
                 log_info("Network interface NamePolicy= disabled on kernel command line.");
